identity/generate_test.go

```diff
@@ -33,6 +34,17 @@ var _ = Describe("generate", func() {
 		Expect(id.Target).Should(Equal("user"))
 	})
 
+	It("generates ordered JSON", func() {
+		identityB64, _ := Create(appConfig, "userID")
+		identityJson, _ := base64.StdEncoding.DecodeString(*identityB64)
+
+		var objmap map[string]interface{}
+		_ = json.Unmarshal(identityJson, &objmap)
+		orderedJson, _ := json.Marshal(objmap)
```

Comment:
I was expecting a test for b64json.Encode that you changed in this commit, not raw json